src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -7,14 +7,15 @@ import { KEYS } from '../../constants';
 
 import Pill from './Pill';
 import SuggestedPillsRow from './SuggestedPillsRow';
-import type { SelectedOptions, SuggestedPillsFilter } from './flowTypes';
+import type { SelectedPills, SuggestedPillsFilter } from './flowTypes';
```

Comment:
I am always confuse on this selectedPills naming.. want to make sure I understand this correctly.. selectedPill means the pill that's currently in the pill selector?

Reply:
Yeah, Pill at the moment is { text, value } while Option is { displayText, value }.
Eventually everything should move to Option. Option will then be a Pill or a option within the drop down.